doc/cabal-commands.rst

```diff
@@ -1076,6 +1076,10 @@ to Hackage.
     documentation for a published package (and not a candidate), add
     ``--publish``.
 
+.. option:: -t, --token
+
+    Your Hackage authentication token.
```

Comment:
Is there any page we can link? (Ideally something like “How to generate and use auth tokens on hackage”)

Reply:
You can create and delete tokens on your own manage account page. The following link will prompt you to log in to Hackage where it tells you how to create and delete tokens: https://hackage.haskell.org/users/account-management. I've looked through Hackage but I haven't found any mention of token authentication outside of the logged in account management page.  Would it be okay linking to a page that requires a login in this situation?

Reply:
I would say so, something like

```
Your Hackage authentication token. See XYZ on how to create/delete tokens.
```

Thanks!